docs/source/using-discovery/transferringdata.md

```diff
@@ -1,44 +1,124 @@
 
-Where `<filename>` is the name of the file you want to transfer and `<yourusername>` is your Northeastern username.
+You can also use rsync to copy data from different directories on the HPC:
+
+:::{code-block} bash
+srun --partition=short --nodes=1 --ntasks=1 --time=01:05:00 --constraint=ib --pty /bin/bash
+rsync -av /scratch/<username>/source_folder /home/<username>/destination_folder
+:::
 
-### Using SSHFS
+::::
+::::{tab-item} sbatch
+
+You can use an sbatch job to complete data transfers by submitting the job to the HPC queue. An example of using `rsync` through an sbatch script is as follows:
+
+:::{code-block} bash
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --ntasks=2
+#SBATCH --time=0:05:00
+#SBATCH --job-name=DataTransfer
+#SBATCH --mem=2G
+#SBATCH --partition=short
+#SBATCH --constraint=ib
+#SBATCH -o %j.out
+#SBATCH -e %j.err
+
+rsync -av /scratch/<username>/source_folder /home/<username>/destination_folder
+:::
 
-If you want to use `sshfs`, you will need to use it with the dedicated transfer node `xfer.discovery.neu.edu`. It will not work on the login or compute nodes.
+where we are transfering the data from `source_folder` to the `destination_folder`.
+
+::::
+::::{tab-item} SSHFS
+
+If you want to use `sshfs`, you will need to use it with the dedicated transfer node `xfer.discovery.neu.edu`. It will not work on the login or compute nodes. On a Mac, you will also have to install macFUSE and sshfs (please refer to [macFUSE]) to use the `sshfs` command.
 
 Use this syntax to perform file transfers with `sshfs`:
 
+:::{code-block} bash
+sshfs <username>@xfer.discovery.neu.edu:</your/remote/path> <your/local/path> -<options>
 :::
-sshfs <yourusername>@xfer.discovery.neu.edu:</your/remote/path> <your/local/path> -<options>
+
+For example, this will mount a directory in your `/scratch` named `test-data` to a local directory on your machine `~/mount_point`:
+
+:::{code-block} bash
+sshfs <username>@xfer.discovery.neu.edu:/scratch/<username>/test-data ~/mount_point
 :::
 
-## Data transfer node using Windows
+where you can interact with the directory from your GUI or using the terminal to perform tasks on it.
+::::
+:::::
 
-You can use MobaXterm to transfer data to and from Discovery. You can also use a file transfer program, such as FileZilla.
-The Files menu in Open onDemand (OOD) also allows you to transfer files. See [Using OOD's File Explorer](../using-ood/fileexplore.md) for more information.
+## Transfer via GUI Application
 
-**To transfer data using MobaXterm:**
+:::::{tab-set}
+::::{tab-item} OOD's File Explorer
+
+You can use OOD's File Explorer application to transfer data from different directories on the HPC and also to transfer data to and from your local machine to the HPC. For more information to complete this please see {ref}`file-explorer`.
+::::
+::::{tab-item} MobaXterm
+
+You can use MobaXterm to transfer data to and from the HPC.
```

Comment:
Link for download?

Reply:
I added the link to MobaXterm as well. Thank you!!